front/src/features/tables/components/ColumnContentTags.tsx

```diff
@@ -1,12 +1,32 @@
+import { TagPopup } from "@/features/tables/components/TagPopup";
 import { Tag } from "@prisma/client";
+import { useState } from "react";
 import { TagLabel } from "../../../components/atoms/TagLabel";
+import { concatAndSortTags, isTagInArray } from "../helpers/tagArrayManagement";
 
-export const ColumnContentTags = ({ tags }: { tags: Tag[] }) => {
+export const ColumnContentTags = ({
+    tags,
+    logId,
+}: {
+    tags: Tag[];
+    logId: string;
+}) => {
+    const [tagArray, setTagArray] = useState<Tag[]>(tags);
+    const addTag = (tag: Tag) => {
```

Comment:
so this is just setting it locally for now?

Reply:
Yes, the idea is to update the database and the UI at the same time so that the user sees the new tag on his log without having to refresh the whole page. To do that I had to store the tags in this state.